src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,54 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SEPARATOR = ",";
+    private static final int WORD_INDEX = 0;
+    private static final int NUMBER_INDEX = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
It is necessary to break the logic into 3 methods. The getStatistic() method should only consist of calling these methods:
String dataFromFile = readFile(fromFileName);
String report = createReport(dataFromFile);
writeReportToFile(report, toFileName);